hi-ml-histopathology/src/histopathology/datamodules/panda_module.py

```diff
@@ -32,3 +34,63 @@ def get_splits(self) -> Tuple[PandaTilesDataset, PandaTilesDataset, PandaTilesDa
         return (PandaTilesDataset(self.root_path, dataset_df=splits.train),
                 PandaTilesDataset(self.root_path, dataset_df=splits.val),
                 PandaTilesDataset(self.root_path, dataset_df=splits.test))
+
+
+class SubPandaTilesDataModule(TilesDataModule):
+    """ subPandaTilesDataModule is the child class of PandaDataModule specific to PANDA dataset
+    Method get_splits() returns the train, val, test splits from a subset of the PANDA dataset specified by
+    train/validation dataframes.
+    """
+
+    def __init__(self, train_csv: Union[str, Path], val_csv: Union[str, Path], **kwargs: Any) -> None:
+        self.train_csv = train_csv
+        self.val_csv = val_csv
+        super().__init__(**kwargs)
+
+    def get_splits(self) -> Tuple[PandaTilesDataset, PandaTilesDataset, PandaTilesDataset]:
+        return (
+            PandaTilesDataset(self.root_path, self.train_csv),
+            PandaTilesDataset(self.root_path, self.val_csv),
+            PandaTilesDataset(self.root_path, self.val_csv),
```

Comment:
we're returning two times the validation set, this is prone to errors down the line, should we have a test file? or return an empty dataset?